languages/react-native/ReactNative.podspec

```
@@ -0,0 +1,25 @@
+package = JSON.parse(File.read(File.join(__dir__, "package.json")))
+
+Pod::Spec.new do |s|
+  s.name         = "ReactNative"
+  s.version      = package["version"]
+  s.summary      = package["description"]
+  s.homepage     = package["homepage"]
+  s.license      = package["license"]
+  s.authors      = package["author"]
+
+  s.platforms    = { :ios => min_ios_version_supported }
+  s.source       = { :git => "https://github.com/EATSTEAK/rusaint.git", :tag => "#{s.version}" }
```

RuboCop (1.75.5)

[convention] Prefer `to_s` over string interpolation.

(Style/RedundantInterpolation)